From 797247c9a319bec7adf860df2597b89edbe36e18 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Fri, 5 Sep 2014 18:55:46 -0400
Subject: [kernel] kexec: Typo s/the/then/

Message-id: <1409943373-12669-12-git-send-email-vgoyal@redhat.com>
Patchwork-id: 91517
O-Subject: [RHEL7.1 PATCH 11/38] kexec: Typo s/the/then/
Bugzilla: 852066
RH-Acked-by: Dave Anderson <anderson@redhat.com>
RH-Acked-by: Baoquan He <bhe@redhat.com>
RH-Acked-by: Dave Young <dyoung@redhat.com>
RH-Acked-by: Chao WANG <chaowang@redhat.com>

BZ:852066

commit 002ace782cc8eef178808cc2b36853750154fe19
Author: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Sun, 15 Sep 2013 11:35:37 +0200

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/kernel/kexec.c b/kernel/kexec.c
index e89d3e5..27f4d19 100644
--- a/kernel/kexec.c
+++ b/kernel/kexec.c
@@ -927,7 +927,7 @@ static int kimage_load_segment(struct kimage *image,
  *   reinitialize them.
  *
  * - A machine specific part that includes the syscall number
- *   and the copies the image to it's final destination.  And
+ *   and then copies the image to it's final destination.  And
  *   jumps into the image at entry.
  *
  * kexec does not sync, or unmount filesystems so if you need
-- 
1.7.1